### PR TITLE
docs: split CONTRIBUTING.md into focused development and PR docs

### PR DESCRIPTION
**Pull request by DysektAI**
## Summary

- Split `.github/CONTRIBUTING.md` (was 331 lines) into a 75-line entry point plus two focused docs:
  - `docs/contributing/development.md` — detailed dev setup, coding standards, common tasks, debugging, commit conventions
  - `docs/contributing/pull-requests.md` — branching strategy, PR requirements, PR template field explanation, review process
- Removed the duplicated "Issue Types & Label System" and "Project Board" sections from CONTRIBUTING.md (now just links to `.github/LABELS.md` and `.github/PROJECT_BOARD.md`).
- Removed the detailed "Guidelines for AI Coding Agents" section from CONTRIBUTING.md (lives in AGENTS.md only). Kept a one-line agent-use notice pointing to AGENTS.md.
- Updated `docs/README.md` routing table and document map with the two new docs.
- Updated `AGENTS.md` Deeper References to reflect the new ownership model.

#### Test plan
- [x] Prettier passes on all 5 changed markdown files
- [x] All relative links in new/modified files resolve (scripted check)
- [x] Husky pre-commit hook passed
- [ ] CI `link-check` passes
- [ ] CI `format:check` passes

---

<a href="https://app.devin.ai/review/tarkovtracker-org/tarkovtracker/pull/579" target="_blank">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://static.devin.ai/assets/gh-open-in-devin-review-dark.svg?v=1">
    Open in Devin Review
  </picture>
</a>

<a href="https://cubic.dev/pr/tarkovtracker-org/TarkovTracker/pull/579?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

___

## **CodeAnt-AI Description**
**Split contributor guidance into focused setup and pull request docs**

### What Changed
- Moved local development setup, coding standards, common tasks, debugging, and commit rules into a dedicated development guide
- Moved branching, PR requirements, template fields, review steps, and submission checklist into a dedicated pull request guide
- Shortened the main contributing page to act as a summary and entry point, with links to the new guides
- Updated the docs index and agent references so contributors can find the right guide quickly

### Impact
`✅ Faster onboarding for new contributors`
`✅ Easier pull request preparation`
`✅ Clearer path to the right contributor guide`
<details><summary><strong>💡 Usage Guide</strong></summary>

### Checking Your Pull Request
Every time you make a pull request, our system automatically looks through it. We check for security issues, mistakes in how you're setting up your infrastructure, and common code problems. We do this to make sure your changes are solid and won't cause any trouble later.

### Talking to CodeAnt AI
Got a question or need a hand with something in your pull request? You can easily get in touch with CodeAnt AI right here. Just type the following in a comment on your pull request, and replace "Your question here" with whatever you want to ask:
<pre>
<code>@codeant-ai ask: Your question here</code>
</pre>
This lets you have a chat with CodeAnt AI about your pull request, making it easier to understand and improve your code.

#### Example
<pre>
<code>@codeant-ai ask: Can you suggest a safer alternative to storing this secret?</code>
</pre>

### Preserve Org Learnings with CodeAnt
You can record team preferences so CodeAnt AI applies them in future reviews. Reply directly to the specific CodeAnt AI suggestion (in the same thread) and replace "Your feedback here" with your input:
<pre>
<code>@codeant-ai: Your feedback here</code>
</pre>
This helps CodeAnt AI learn and adapt to your team's coding style and standards.

#### Example
<pre>
<code>@codeant-ai: Do not flag unused imports.</code>
</pre>

### Retrigger review
Ask CodeAnt AI to review the PR again, by typing:
<pre>
<code>@codeant-ai: review</code>
</pre>

### Check Your Repository Health
To analyze the health of your code repository, visit our dashboard at [https://app.codeant.ai](https://app.codeant.ai). This tool helps you identify potential issues and areas for improvement in your codebase, ensuring your repository maintains high standards of code health.

</details>

<details open><summary><h3>Confidence Score: 5/5</h3></summary>

Pure documentation reorganisation with no code changes; all cross-file links were verified to resolve.

Every file touched is documentation only. All relative links across the five changed files resolve to existing paths. Content is consistent across the entry point and the two new focused guides. The nav-link target correction in development.md (DrawerLinks.vue vs NavDrawer.vue) reflects the actual codebase structure.

No files require special attention.
</details>

<details><summary><h3>Important Files Changed</h3></summary>

| Filename | Overview |
|----------|----------|
| .github/CONTRIBUTING.md | Trimmed from 331 to 76 lines; now acts as an entry point with summaries and links to the new focused guides. All relative links verified. |
| docs/contributing/development.md | New file covering setup, coding standards, common tasks, debugging, and commit conventions. Silently corrects the nav-link target to DrawerLinks.vue (correct) from the old NavDrawer.vue reference. All links resolve. |
| docs/contributing/pull-requests.md | New file covering branching strategy, pre-submit checklist, PR template fields, and review process. 'Finding Work' section removed per previous review thread. All links resolve. |
| docs/README.md | Adds two new routing rows and document-map entries for the new contributing guides; no broken links introduced. |
| AGENTS.md | Deeper References section updated to split the single CONTRIBUTING.md entry into three entries reflecting the new ownership model. Minimal, correct change. |

</details>

<details><summary><h3>Flowchart</h3></summary>

```mermaid
%%{init: {'theme': 'neutral'}}%%
flowchart TD
    A[".github/CONTRIBUTING.md\n(entry point, 76 lines)"] --> B["docs/contributing/development.md\nSetup · Standards · Tasks · Debug · Commits"]
    A --> C["docs/contributing/pull-requests.md\nBranching · Checklist · Template · Review"]
    A --> D[".github/LABELS.md"]
    A --> E[".github/PROJECT_BOARD.md"]
    B --> F["AGENTS.md\n(canonical coding standards)"]
    C --> F
    G["docs/README.md\n(routing table)"] --> A
    G --> B
    G --> C
    H["AGENTS.md\nDeeper References"] --> A
    H --> B
    H --> C
```
</details>

<sub>Reviews (3): Last reviewed commit: ["fix: address re-review findings on CONTR..."](https://github.com/tarkovtracker-org/tarkovtracker/commit/0cff29bfc3836e8af76360974c70964f356bd0bc) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=46172219)</sub>

**Context used:**

- Context used - AGENTS.md ([source](https://app.greptile.com/dysektai/github/tarkovtracker-org/tarkovtracker/-/custom-context?memory=635c24fb-71e5-43a4-9979-74405b725e95))

### CODE REVIEW

File: docs/contributing/development.md
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/development.md:1]: Acknowledged — thanks for verifying all relative links resolve. Resolving.

File: docs/contributing/pull-requests.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- **Area(s) Affected** — mark all that apply: Frontend, Backend, Tasks/Quests, Team Features,
+  Hideout, Maps, Traders, API, i18n/Translations, or Other.
+- **Related Issues** — link related issues using keywords (`Fixes #123`, `Closes #456`,
+  `Related to #789`).
+- **Testing** — mark how you tested (locally, production-like environment, unit tests, manual) and
+  describe your test plan.
+- **Screenshots/Videos** — add screenshots or videos for UI changes.
+- **Checklist** — confirm your code follows conventions, you self-reviewed, documentation is updated,
+  no new warnings/errors, tests pass, and breaking changes are checked.
+- **Additional Notes** — anything reviewers should know.
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/pull-requests.md:75]: Acknowledged — note that PR #577 (template changes) will require updating the template field descriptions here. That PR should merge first. Resolving.

File: docs/contributing/pull-requests.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Finding Work
+
+- Check existing issues before creating new ones
+- Use appropriate issue templates (Bug Report, Feature Request, Enhancement, Data Issue)
+- Comment on an issue to express interest and wait for maintainer assignment to avoid duplicate work
+- Issues labeled `good-first-issue` are great for newcomers
+- See [`../../.github/LABELS.md`](../../.github/LABELS.md) for the complete label reference and
+  [`../../.github/PROJECT_BOARD.md`](../../.github/PROJECT_BOARD.md) for project-board workflow
+  states
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/pull-requests.md]: Fixed in 73b7528c. Removed the "Finding Work" section from pull-requests.md and replaced with a link to CONTRIBUTING.md. Resolving.

File: docs/contributing/development.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+- Write tests for new features
+- Keep tests deterministic
+- Mock network/Supabase calls
+- Run `pnpm test` before submitting
+
+## Common Tasks
+
+- **Add a feature:** create a slice in `app/features/`, add a route in `app/pages/`, and a nav link
+  in `app/shell/NavDrawer.vue`.
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/development.md]: Fixed in 73b7528c. Common-task recipe now points to app/features/drawer/DrawerLinks.vue. Resolving.

File: docs/contributing/development.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+## Prerequisites
+
+- **Node.js** >= 24.12.0
+- **pnpm** 11.14.0 (via Corepack; matches `packageManager` in `package.json`)
+- **Git**
+
+## Installation & Environment Setup
+
+1. **Fork the repository** and clone your fork locally
+2. **Install dependencies**: `corepack enable && pnpm install` (Corepack resolves the pnpm version
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/development.md]: Fixed in 73b7528c. Combined install + setup into one step: `corepack enable && pnpm run setup`. Resolving.

File: docs/contributing/pull-requests.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  describe your test plan.
+- **Screenshots/Videos** — add screenshots or videos for UI changes.
+- **Checklist** — confirm your code follows conventions, you self-reviewed, documentation is updated,
+  no new warnings/errors, tests pass, and breaking changes are checked.
+- **Additional Notes** — anything reviewers should know.
+
+## Review Process
+
+1. Maintainers will review your PR
+2. Address any feedback or requested changes
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/pull-requests.md]: Fixed in 73b7528c. Review process now states every thread must have an explicit disposition before merge, with a link to AGENTS.md PR Review Gate. Resolving.

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -1,331 +1,75 @@
...
-- If a task has multiple valid interpretations, ask which approach is preferred
-- When uncertain about scope, confirm boundaries before making changes
-- It's better to ask one clarifying question than to redo work
+## Pull Request Process (Summary)
 
-### Communicate Proactively
+1. Create a branch: `git checkout -b type/short-description`
+   (`fix/`, `feat/`, `enhance/`, `refactor/`, `docs/`, `chore/`)
+2. Make your changes following the coding standards
+3. Self-review, test (`pnpm run lint`, `pnpm run typecheck`, `pnpm test`), and update docs
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply .github/CONTRIBUTING.md]: Fixed in 73b7528c. CONTRIBUTING.md now says "run the smallest relevant validation" with conditional commands. Resolving.

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -1,331 +1,75 @@
...
-- `.claude/CLAUDE.md` is a thin shim that imports `AGENTS.md` for Claude Code (moved from root to reduce clutter).
-- `GEMINI.md` is intentionally not tracked. If you use Gemini CLI, point it at `AGENTS.md` with `.gemini/settings.json`:
-
-```json
-{
-  "context": {
-    "fileName": ["AGENTS.md"]
-  }
-}
+corepack enable && pnpm install
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply .github/CONTRIBUTING.md]: Fixed in 73b7528c. Quick-start now uses `corepack enable && pnpm run setup` (one step). Resolving.

File: docs/contributing/development.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
+# Development Setup & Coding Standards
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/development.md:1]: Fixed in 73b7528c. Coding standards section now links to AGENTS.md as canonical source with only key reminders, no duplicated rules. Resolving.

File: docs/contributing/development.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+- Log errors with `logger` from `@/utils/logger`
+- Provide user-friendly error messages
+- Handle edge cases gracefully
+
+### Testing
+
+- Write tests for new features
+- Keep tests deterministic
+- Mock network/Supabase calls
+- Run `pnpm test` before submitting
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/development.md]: Fixed in 73b7528c. Testing instruction now says "only if your change touches executable code that could break tests" and notes docs-only PRs do not need tests. Resolving.

File: docs/contributing/pull-requests.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+3. Once approved, maintainers will merge
+4. Your contribution will be in the next release!
+
+## Finding Work
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/pull-requests.md]: Fixed in 73b7528c. Finding Work section removed; replaced with link to CONTRIBUTING.md. Resolving.

File: docs/contributing/pull-requests.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+   - Test in both PvP and PvE modes if relevant
+
+3. **Update documentation**
+   - Update README if adding features
+   - Update [`../../AGENTS.md`](../../AGENTS.md) if changing architecture or repo-wide agent guidance
+
+## PR Requirements
+
+- Title and commits follow [Conventional Commits](https://www.conventionalcommits.org/) (e.g.
+  `feat(tasks): add objective filter`) — enforced by commitlint
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/pull-requests.md]: Fixed in 73b7528c. Changed "Title and commits" to "Commits" — commitlint enforces commit messages, not PR titles. Resolving.

File: docs/contributing/pull-requests.md [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+   - Remove debug logs and commented code
+   - Check for typos and formatting
+   - Ensure no secrets or credentials
+
+2. **Test thoroughly**
+   - Test manually in browser
+   - Run `pnpm run lint` (must pass with zero warnings)
+   - Run `pnpm run typecheck`
+   - Run `pnpm test` if your change touches executable code
+   - Test in both PvP and PvE modes if relevant
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/pull-requests.md]: Fixed in 73b7528c. Validation section now makes each command conditional on the affected content, with docs-only PRs explicitly exempt. Resolving.

File: docs/contributing/pull-requests.md [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+  no new warnings/errors, tests pass, and breaking changes are checked.
+- **Additional Notes** — anything reviewers should know.
+
+## Review Process
+
+1. Maintainers and automated review tools review your PR
+2. Address all feedback on the same PR branch — do not open follow-up PRs for in-scope feedback
+3. Every review thread must have an explicit disposition (fixed, rejected with rationale, or
+   deferred to a tracked issue) before merge
+4. Once all threads are resolved and CI is green, a maintainer merges
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/pull-requests.md]: Fixed in 0cff29bf. Merge criteria now explicitly includes "every inline review thread and every top-level/review-summary comment" and "all reviews complete". Resolving.

File: .github/CONTRIBUTING.md [hunk 14]
```diff
@@ -1,331 +1,75 @@
...
-- When uncertain about scope, confirm boundaries before making changes
-- It's better to ask one clarifying question than to redo work
+## Pull Request Process (Summary)
 
-### Communicate Proactively
+1. Create a branch: `git checkout -b type/short-description`
+   (`fix/`, `feat/`, `enhance/`, `refactor/`, `docs/`, `chore/`)
+2. Make your changes following the coding standards
+3. Self-review, run the smallest relevant validation (`pnpm run lint` for code, `pnpm run typecheck`
+   for TS, `pnpm test` for executable code), and update docs
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply .github/CONTRIBUTING.md]: Fixed in 0cff29bf. Added `pnpm --filter api-gateway run typecheck` for Workers changes. Resolving.

File: docs/contributing/development.md [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+   reference: [`../runbook.md`](../runbook.md) and [`../ARCHITECTURE.md`](../ARCHITECTURE.md).
+3. **Start dev server**: `pnpm run dev` (serves on `localhost:3000`)
+4. **Read [`../../AGENTS.md`](../../AGENTS.md)** for detailed development guidelines
+
+> Most features work without Supabase configured; auth and sync are simply disabled.
+
+## Coding Standards
+
+Coding standards are documented in [`../../AGENTS.md`](../../AGENTS.md) (Coding Conventions,
+Nuxt/Vue Rules, TypeScript, Error Handling, Localization sections). That file is the canonical
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/contributing/development.md]: Fixed in 0cff29bf. Changed "Nuxt/Vue Rules" to "Nuxt / Vue Rules" to match the exact AGENTS.md heading. Resolving.
